src/log.py

```diff
@@ -0,0 +1,63 @@
+"""
+log
+
+By: Norma Chang
+"""
+import json
+import os.path
+import time
+
+
+class Log:
+    """
+    Logging class.
+
+    Logs data from vehicle.
+    """
+    def __init__(self, vehicle_name):
+        self.file_name = ""
+        self.class_name = ""
+        self._start_time = time.strftime('%b %d %Y %H:%M:%S', time.gmtime())
+        self._start_time = self._start_time.replace(" ", "").replace(":", "")
+        if vehicle_name.lower() == 'follower':
```

Comment:
Like I mentioned yesterday in class, it doesn't matter what type of vehicle it is. You only use this for the file name. Why can't they pass in anything they want?

Reply:
Made it general so they can pass in anything for a file name